.eslintrc.js

```diff
@@ -4,9 +4,7 @@ module.exports = {
     jest: true
```

Comment:
Please remove all unnecessary files from your PR, except for the file with tests